### PR TITLE
Fixed byte server messages

### PR DESCRIPTION
**Pull request by adamselevan**
Sequence number was not casted correctly.

### CODE REVIEW

**Comment by dansantorojr** [comment]
To be clear, we're fixing the issue with the sequence number length here, right? Assembler still looks like it's sending strings, just want to make sure that was accurate for what we're trying to accomplish with this pull request.

File: server/gateway.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ var Event      = require('./models/Event');
 var WS 			= require('./websockets');
 
 // Constants
-const DEFAULT_SERIAL_PORT = '/dev/tty.usbserial-AE00BUMD';
+const DEFAULT_SERIAL_PORT = '/dev/ttty.usbserial-AM017Y3E';
```
Comment (kamladi) [inline server/gateway.js]: Are you sure there's supposed to be an extra 't' here?

Comment (dansantorojr) [inline server/gateway.js]: ooooo I didn't know you can comment on specific lines!

**Comment by adamselevan** [comment]
Correct. Assembler still sends strings to server. Discussed this issue with Kedar yesterday. I was not able to format data with printf() to his liking. printf("%02X") was noting adding 0's for some reason.  I am moving on to other items so that we can test other pieces. Didn't want to hold you guys up, I will readdress this issue later today.

File: server/gateway.js (hunk 1 above)
Comment (adamselevan) [inline server/gateway.js]: Yea that extra 't' should not be there...

**Comment by dansantorojr** [comment]
In that case, looks good to me!

**Comment by dansantorojr** [comment]
Obviously, that possible extra "t" isn't a huge issue

**Pull request merged**